lib/shopify_app/utils.rb

```diff
@@ -27,5 +27,41 @@ def self.shop_login_url(shop:, host:, return_to:)
 
       url.to_s
     end
+
+    module Logger
```

Comment:
I think 'Utils' is not a helpful grouping so I would suggest moving this up a level to `ShopifyApp::Logger`.